docs/getting-started.md

```diff
@@ -33,6 +33,14 @@ cd a12n-server
 npm install
 ```
 
+Setup the environment:
+
+```sh
+npm run prepare
+```
+
+This will create a `.env` file with a private JWT for local development of OAuth2/OIDC flows.
+
```

Comment:
This probably needs a bit of work first. `prepare` is a bit of a misnomer, because the script actually is the 'dev server' script, also mentioned in `README.md`, so it does more than just preparing, it actually starts the server with default (dev) configuration. But I agree that we should make it easier to get started.

I'm thinking we really 4 separate documents for each use-case
 
1. dev server for testing. (`npx script`)
2. from source (requires a checkout, `npm i`, etc).
3. in production

And possibly also :cry: 

4. With docker compose

Reply:
yeah so far, getting-started is for developers who are using this package, not for contributors right?

Reply:
When you say "from source", you really mean it's the doc for contributors right?

Reply:
yes!